Makefile

```diff
@@ -24,6 +24,12 @@ bins: $(BINS)
 build:
 	$(GO) build ./...
 
+generate: stringer
+	$(GO) generate ./...
+
+stringer:
+	$(GO) install golang.org/x/tools/cmd/stringer 
```

Comment:
can you add this to [`deveps-install`](https://github.com/ovrclk/akash/blob/2859bc805774ddfded6336d8871bb03505832079/Makefile#L135-L138) instead of making a new target?

Also please add `golang.org/x/tools/cmd/stringer` to [`tools.go`](https://github.com/ovrclk/akash/blob/2859bc805774ddfded6336d8871bb03505832079/tools.go#L7-L9)